meta-networking/recipes-connectivity/sshpass/sshpass_1.09.bb

```diff
@@ -1,7 +1,7 @@
 DESCRIPTION = "Non-interactive ssh password auth"
 HOMEPAGE = "http://sshpass.sourceforge.net/"
 SECTION = "console/network"
-LICENSE = "GPLv2"
+LICENSE = "GPL-2.0-only"
```

Comment:
The older license string caused the following warning:

`WARNING: sshpass-1.09-r0 do_package_qa: QA Issue: Recipe LICENSE includes obsolete licenses GPLv2 [obsolete-license]`

Changing the string to the newer value fixes the issue.

Reply:
Can you please not include the license change in this PR seeing that it's just a warning and not an error? I'd rather we have a separate PR for it where we review it and try to send it upstream first before merging it.

Reply:
Btw, the commit message for this change also has some issues:
- Not signed off
- File name not mentioned in first line
- No description in commit message why this change is being made